docs/linux/fs.md

```diff
@@ -419,89 +419,91 @@ Birth: -
-entirely of sequential zones cannot be used with *f2fs* as a standalone device
-and require a multi-device setup to place metadata blocks on a randomly
-writable storage. *f2fs* supports multi-device setup where multiple block device
-address spaces are linearly concatenated to form a logically larger block
-device. The [*dm-linear*](./dm.md#dm-linear) device mapper target can also be used
-to create a logical device composed of conventional zones and sequential zones
-suitable for *f2fs*.
+Zoned block device support was added to *f2fs* with kernel 4.10. Because *f2fs*
+uses a metadata-block on-disk format with fixed-block location, only zoned
+block devices that include conventional zones are supported. Zoned devices
+composed entirely of sequential zones cannot be used with *f2fs* as a
+standalone device and they require a multi-device setup in order to place
+metadata blocks on randomly writable storage. *f2fs* supports multi-device
+setup where multiple block device address spaces are linearly concatenated to
+form a logically larger block device. The [*dm-linear*](./dm.md#dm-linear)
+device mapper target can also be used to create a logical device that is
+composed of both conventional zones and sequential zones suitable for *f2fs*.
 
 *f2fs* zoned block device support was achieved using the following principles.
 
-1. **Section Alignment** In *f2fs*, a section is a group of fixed size
+1. **Section Alignment** In *f2fs*, a section is a group of fixed-size
    segments (2 MB). The number of segments in a section is determined to match
-   the zoned device zone size. For instance, with a 256 MB zone size, a section
+   the zoned device zone size. For example: with a 256 MB zone size, a section
    contains 128 segments of 2MB.
-2. **Forced LFS mode** By default, *f2fs* tries to optimize block allocation to
-   avoid excessive append write by allowing some random writes within segments.
-   The LFS mode forces sequential writes to segments and the sequential use of
-   segments within sections, resulting in full compliance with zoned block
-   devices write constraint.
-3. **Zone reset as discard operation** Block *discard* (or *trim*) used to
-   indicate to a device that a block or range of blocks are no longer in use is
-   replaced with execution of a zone write pointer reset command when all blocks
-   of all segments of a section are free, allowing the section to be reused.
-
-Compared to a solution using the *dm-zoned* device mapper target, performance
-of *f2fs* on zoned devices does not suffer from zone reclaim overhead as writes
-are always sequential and do not require on-disk temporary buffering. *f2fs*
-garbage collection (segment cleanup) will generate overhead only for workloads
-frequently deleting file or modifying files data.
+2. **Forced LFS mode** By default, *f2fs* tries to optimize block allocation (in   order to avoid excessive append write) by allowing some random writes within 
```

Comment:
It looks like line wrapping changed: this line is much longer than it was. Maybe add a line break before the "(" ?
Hard to be sure with the github web I/F though. Would need to vi the file with patch applied :)

Reply:
Here we have the fruits of my struggle against the limits of my command of vim. I had thought with "gq" to be able to indent all of these lines so that they were congruent with the ordered list in which they appear, but instead I got this mess of a long line.

I've repaired it.

And now, after four days of having no internet, I push this back to you for merging.